src/main/fc/stats.h

```diff
@@ -0,0 +1,15 @@
+
+#pragma once
+#ifdef USE_STATS
```

Comment:
There should not be any conditionals in here. Also, using 'placebo' functions make the code using this hard to understand - please make the function calls conditional instead.

Reply:
This way I would need to add ifdef-s. I don't like ifdefs :(
But I can change of course...